pepper-apis/src/main/resources/db-changes/schema/DDP-5442-auth0-log-event.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="evgeniipr" id="20210113-create-auth0-log-event-table">
+        <createTable tableName="auth0_log_event">
+            <column name="auth0_log_event_id" type="bigint" autoIncrement="true" startWith="1">
+                <constraints primaryKey="true" primaryKeyName="auth0_log_event_pk"/>
+            </column>
+            <column name="tenant" type="varchar(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="log_id" type="varchar(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date" type="datetime(6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="type" type="varchar(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="client_id" type="varchar(200)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="connection_id" type="varchar(200)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="user_id" type="varchar(200)">
```

Comment:
`user_id` might be misunderstood to be a reference to `user.user_id`.  Can you change this to something like `auth0_user_id`?

Reply:
Fixed. Renamed to auth0_log_event_user_id. Also renamed couple of other fields.

Reply:
In my opinion, this name is a little too verbose... I'm in favor of Zim's suggestion of `auth0_user_id`. It's understandable and consistent with how this piece of data is named in other tables.

Reply:
Renamed it to auth0_user_id.